src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/parameters/Base64EncodedStringParameterValue/Base64EncodedStringParameterValue.jelly

```diff
@@ -27,6 +27,9 @@ THE SOFTWARE.
         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
         <f:entry title="${it.name}" description="${it.description}">
+            <div name="parameter" description="${it.description}">
```

Comment:
@rinrinne Since renamed jelly file would never be loaded in Gerrit Trigger, is it better if the name is changed back to *value.jelly*? I fear this would potentially cause future bugs.

No, sorry, I don't know any plugins that implements this class either. I need to google it to find out too.